sci-visualization/spyview/spyview-20150124.ebuild

```diff
@@ -29,6 +29,8 @@ DEPEND="${COMMON_DEPEND}
 RDEPEND="${COMMON_DEPEND}
 	sci-visualization/gnuplot"
 
+PATCHES=( ${FILESDIR}/${P}-gnuplot_interface_fix.patch )
+
```

Comment:
try (might fail though) `append-libs` instead of `append-ldflags`, as `LDFLAGS` is not the right variable for `-L` options (might fail though, as the build system needs to respect `LIBS`)